src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -300,13 +303,13 @@ const JobHistory = _.flow(
               }
             },
             {
-              size: { basis: 170, grow: 0 },
+              size: { basis: 150, grow: 0 },
```

Comment:
Am I reading this right? You made this field slightly narrower at the same time as making the font smaller?

I thought a `grow: 0` field would never resize, so I'm confused why this column was ever getting too narrow (or perhaps the problem was that it was never wide enough?)

Reply:
Because I made the font smaller, I thought that it was OK to also make the field smaller-- I decided to make the size consistent with the submission ID field next to it (also ran this by Brian). When the date wraps to 3 lines it still looks OK because of the additional vertical spacing.

There are separate variables for grow and shrink. This field will not grow, but it can shrink. In contrast, the submission ID field is not allowed to shrink because if it does, the ID starts rendering outside of the cell because it can only break at specific places.